### PR TITLE
fixed the output format

### PR DESCRIPTION
**Pull request by shlao**
Hi, I submit a pr to fix the wrong output format

### CODE REVIEW

File: docs/user/troubleshooting.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ The installer uses the [cluster-version-operator] to create all the components o
 1. Inspecting the `ClusterVersion` object.
 
     ```console
-    $ oc --config=${INSTALL_DIR}/auth/kubeconfig get clusterversion -oyaml
+    $ oc --config=${INSTALL_DIR}/auth/kubeconfig get clusterversion -o yaml
```
Comment (abhinavdahiya) [inline docs/user/troubleshooting.md]:
```
$ oc version
Client Version: version.Info{Major:"4", Minor:"0+", GitVersion:"v4.0.0-alpha.0+fbfe4e9-2052", GitCommit:"fbfe4e9397", GitTreeState:"", BuildDate:"2019-04-29T20:08:12Z", GoVersion:"", Compiler:"", Platform:""}
Server Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.4+d8a0281", GitCommit:"d8a0281", GitTreeState:"clean", BuildDate:"2019-04-17T19:10:43Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
```
```console
$ oc get clusterversion -oyaml
apiVersion: v1
items:
- apiVersion: config.openshift.io/v1
  kind: ClusterVersion
  metadata:
    creationTimestamp: "2019-04-17T22:00:22Z"
    generation: 1
    name: version
    resourceVersion: "7357343"
    selfLink: /apis/config.openshift.io/v1/clusterversions/version
    uid: 329bc573-615c-11e9-a4a3-ac1f6b82e174
  spec:
    channel: stable-4.0
    clusterID: 134419c5-e23b-4816-b5f4-544ae57e8ed0
    upstream: https://api.openshift.com/api/upgrades_info/v1/graph
  status:
    availableUpdates: null
    conditions:
    - lastTransitionTime: "2019-04-17T22:12:55Z"
      message: Done applying 4.1.0-0.okd-2019-04-17-214307
      status: "True"
      type: Available
    - lastTransitionTime: "2019-04-17T22:06:01Z"
      status: "False"
      type: Failing
    - lastTransitionTime: "2019-04-17T22:12:55Z"
      message: Cluster version is 4.1.0-0.okd-2019-04-17-214307
      status: "False"
      type: Progressing
    - lastTransitionTime: "2019-04-17T22:00:46Z"
      message: 'Unable to retrieve available updates: unknown version 4.1.0-0.okd-2019-04-17-214307'
      reason: RemoteFailed
      status: "False"
      type: RetrievedUpdates
    desired:
      image: registry.svc.ci.openshift.org/origin/release@sha256:9de7b6b1891cb07c616bb3de7a8760b17709cfa44ebf92c3834ac440895b17a2
      version: 4.1.0-0.okd-2019-04-17-214307
    history:
    - completionTime: "2019-04-17T22:12:55Z"
      image: registry.svc.ci.openshift.org/origin/release@sha256:9de7b6b1891cb07c616bb3de7a8760b17709cfa44ebf92c3834ac440895b17a2
      startedTime: "2019-04-17T22:00:46Z"
      state: Completed
      version: 4.1.0-0.okd-2019-04-17-214307
    observedGeneration: 1
    versionHash: vGTYRzQHi2I=
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""
```

it looks like `-oyaml` is a valid invocation.

File: docs/user/troubleshooting.md [hunk 2]
```diff
@@ -241,7 +241,7 @@ The installer fetches the URL for OpenShift console using the [route][route-obje
 1. Check if the console router is `Available` or `Failing`
 
     ```console
-    $ oc --config=${INSTALL_DIR}/auth/kubeconfig get clusteroperator console -oyaml
+    $ oc --config=${INSTALL_DIR}/auth/kubeconfig get clusteroperator console -o yaml
```
Comment (abhinavdahiya) [inline docs/user/troubleshooting.md]: same as https://github.com/openshift/installer/pull/1717/files#r281709669

File: docs/user/troubleshooting.md [hunk 3]
```diff
@@ -452,7 +452,7 @@ From a deployment perspective, the network operator is often the "canary in the
 First, determine that the network configuration exists:
 
 ```console
-$ kubectl get network.config.openshift.io cluster -oyaml
```
Comment (abhinavdahiya) [inline docs/user/troubleshooting.md]:
```console
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.0", GitCommit:"91e7b4fd31fcd3d5f436da26c980becec37ceefe", GitTreeState:"clean", BuildDate:"2018-06-27T20:17:28Z", GoVersion:"go1.10.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.4+d8a0281", GitCommit:"d8a0281", GitTreeState:"clean", BuildDate:"2019-04-17T19:10:43Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
```
```console
$ kubectl get network.config.openshift.io cluster -oyaml
apiVersion: config.openshift.io/v1
kind: Network
metadata:
  creationTimestamp: 2019-04-17T22:00:22Z
  generation: 2
  name: cluster
  resourceVersion: "1637"
  selfLink: /apis/config.openshift.io/v1/networks/cluster
  uid: 325ea9c9-615c-11e9-a4a3-ac1f6b82e174
spec:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  networkType: OpenShiftSDN
  serviceNetwork:
  - 172.30.0.0/16
status:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  clusterNetworkMTU: 1450
  networkType: OpenShiftSDN
  serviceNetwork:
  - 172.30.0.0/16
```

looks like this is valid invocation for kubectl too

**Comment by shlao** [comment]
@abhinavdahiya  Thanks, I update my oc to the latest version and find that the option "-oyaml" is available.
Should I close this pr?

**Comment by abhinavdahiya** [comment]
> @abhinavdahiya Thanks, I update my oc to the latest version and find that the option "-oyaml" is available.
> Should I close this pr?

Yeah, since the invocation is valid I think it's fair to keep it as is. Thanks a lot! Feel free to open PRs for other fixes. Hopefully we'll see more contributions from you.

/close

**Pull request closed without merging**